debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -131,14 +133,20 @@ protected void doCleanup() {
      */
     protected void execute() {
         context.configureLoggingContext("snapshot");
-        logger.info("Starting snapshot");
         final AtomicReference<String> sql = new AtomicReference<>();
         final JdbcConnection mysql = context.jdbc();
         final MySqlSchema schema = context.dbSchema();
         final Filters filters = schema.filters();
         final SourceInfo source = context.source();
         final Clock clock = context.clock();
         final long ts = clock.currentTimeInMillis();
+        logger.info("Starting snapshot for {} with user '{}' and pass '{}' at {}", context.connectionString(), mysql.username(),
+                    mysql.password(), new Date(clock.currentTimeInMillis()));
+        if (logger.isDebugEnabled()) {
+            logServerInformation(sql, mysql);
+            logRolesForCurrentUser(sql, mysql);
+        }
+        
```

Comment:
BTW, I've shifted a bit about what's appropriate for log levels. Since automated monitoring is far more prevalent now, I think it is more useful to include more INFO-level messages that might make it easier for automated rules to look for unexpected conditions. And, a human is less likely to look at the log unless something goes wrong. And in that case, you probably want the log to contain more helpful information.